models/timeswap/arbitrum/timeswap_arbitrum_pools.sql

```diff
@@ -0,0 +1,221 @@
+{{ config(
+    alias = 'pools',
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "timeswap",
+                                \'["raveena15, varunhawk19"]\') }}'
+    )
+}}
+
+SELECT
+    token0_symbol,
+    token1_symbol,
+    token0_address,
+    token1_address,
+    token0_decimals,
+    token1_decimals,
+    strike,
+    maturity,
+    pool_pair,
+    chain,
+    borrow_contract_address,
```

Comment:
You can add a lower() function in all address columns
```suggestion
    lower(borrow_contract_address) as borrow_contract_address,
```

Reply:
Yup, pushed the required changes.